backend/src/controllers/game_controller.ts

```diff
@@ -66,9 +66,8 @@ export class GameController extends Controller {
     }
 
     // save new game
-    const now: number = new Date().getTime();
     const game: Game = new Game();
-    game.startDate = req.body.startDate !== undefined ? toDate(req.body.startDate) || new Date(now + 30 * 1000) : new Date(now + 30 * 1000);
```

Comment:
I do that as we don't use this anymore to determine when the game starts (the player pushing the button is the signal)